source/common/upstream/cds_api_impl.cc

```diff
@@ -1,65 +1,53 @@
 #include "common/upstream/cds_api_impl.h"
 
-#include <chrono>
 #include <string>
-#include <vector>
 
-#include "common/common/assert.h"
-#include "common/config/cds_json.h"
+#include "common/config/subscription_factory.h"
```

Comment:
It doesn't really matter to me but shouldn't most of the includes be transitive? "cds_api_lib" depends on a lot of stuff...